stream_alert_cli/test/results.py

```diff
@@ -249,23 +353,25 @@ def passed(self):
         2) If rules are being tested, all triggered rules match expected rules
         3) If a live test is being performed, all alerts sent to outputs successfully
         """
-        if not self._classified:
+        if not self.classification_status_passed:
             return False
 
-        if self._test_event.get('validate_schema_only'):
+        if not self.rules_run:
```

Comment:
I am pretty confused about what does `rules_run` mean? Does it mean it only runs integration tests against rules, e.g. when we issue the command `python manage.py test rules`? Can you add a comment here?

Reply:
I will rename these properties to be more descriptive and add docblocks